### DISCUSSION TITLE
Run Scalene with Python Package

### DISCUSSION THREAD

**Discussion by GenaNiv** [plasma-umass/scalene#696] [General]
Is it possible to run the  Scalene with installed python package? 
Let's say my source code is a python package, I installed that package and run it with command line:
python3 -m <my package name> --xml-file=<path to xml file>

Thank you!

**Comment by emeryberger** [comment]
Yes, you can invoke it by putting `---` after all the Scalene arguments and before the invocation of the module with `-m`. Here's an example I just tried. Notice the addition of `--profile-all` (to include all packages) and `profile-exclude` (to exclude the Python libraries).

`python3 -m scalene --profile-all --profile-exclude Python --cpu --- -m tqdm --bytes | tail`
